website/docs/docs/integrate/quickstart-semantic-layer.md

```diff
@@ -0,0 +1,296 @@
+
+## Develop and query metrics
+
+You can dynamically develop and query metrics directly in dbt and verify their accuracy *before* running a job in the deployment environment by using the `metrics.calculate` macro and `metrics.develop` macro.
+
+To understand when and how to use the macros above, review [dbt metrics](/docs/building-a-dbt-project/metrics) and make sure you install the [dbt_metrics package](https://github.com/dbt-labs/dbt_metrics) first before using the above macros.
+
+:::info 📌 
+
+Note, you will need access to dbt Cloud and the dbt Semantic Layer from your integrated tool of choice. 
+
+:::
+
+## Run your production job
+
+Once you’ve defined metrics in your dbt project, you can perform a job run in your deployment environment to materialize your metrics. The deployment environment is only supported for the dbt Semantic Layer at this moment. 
+
+1. Go to **Deploy** in the navigation and select **Jobs** to re-run the job with the most recent code in the deployment environment.
+2. Your metric should appear as red nodes in the dbt Cloud IDE and dbt directed acyclic graphs (DAG).
+
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/metrics_red_nodes.png" title="DAG with metrics appearing as red nodes" />
+
+:::info📌 
+
+What’s happening internally?
+
+- Merging the code into your main branch allows dbt Cloud to pull those changes and builds the definition in the manifest produced by the run.
+- Re-running the job in the deployment environment helps materialize the models, which the metrics depend on, in the data platform. It also makes sure that the manifest is up to date.
+- Your dbt Metadata API pulls in the most recent manifest and allows your integration information to extract metadata from it.
+
+:::
+
+## Set up dbt Semantic Layer
+
+:::info 📌
+
+Before you continue with the below steps, you must have a multi tenant dbt Cloud Team or Enterprise plan. Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse dbt metrics in external tools, which requires access to the [Metadata API](/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview).
+
+:::
+
+To query your universally-defined metrics in your integration tool, you need to [set up the dbt Semantic Layer](/docs/integrate/setup-dbt-semantic-layer#set-up-dbt-semantic-layer) in dbt Cloud to connect with your integration tool:
+
+1. In your dbt Cloud account, go to **Account Settings** and then **Service Tokens** to create a new [service account API token](/docs/dbt-cloud/dbt-cloud-api/service-tokens). 
+2. You won't be able to see your token again so we recommend you copy it somewhere safe.
+3. Go to **Deploy** and then **Environment**, and select your **Deployment** environment.
+4. In the upper right side of the page, click **Settings** and then **Edit**.
+5. Select dbt Version 1.2 (latest) or higher.
```

Comment:
At the time we publish this will 1.2 still be the *latest* version.  Might be beneficial to leave "latest" out so we don't have to come back and edit this later.